inc/3-general-info.tex

```diff
@@ -0,0 +1,40 @@
+\section{Общая характеристика состояния атмосферы}
+\begin{frame}{\insertsectionhead}
+    \begin{columns}
+        \begin{column}{.55\linewidth}
+        \footnotesize
+        Выбросы загрязняющих веществ от стационарных и 
+        передвижных источников на территории Саратовской области в 2018 году составили 377,2 тыс. т.
+        По сравнению с предыдущим годом выбросы от стационарных источников уменьшились на 4,6 тыс. т (на 3,8\%), а 
+        выбросы от автотранспорта увеличились на 10,4 тыс. т (на 4,2\%).
+        
+        ~ % Invisible
```

Comment:
What is it?)

Reply:
Just to make empty string and comment to explain what does this char mean

Reply:
`\smallskip`, `\medskip` or `\bigskip`?